tests/fast-integration/test/2-commerce-mock.js

```diff
@@ -24,7 +24,7 @@ const {
 describe("CommerceMock tests", function () {
   this.timeout(10 * 60 * 1000);
   this.slow(5000);
-  const withCentralApplicationGateway = process.env.WITH_CENTRAL_APPLICATION_GATEWAY || false;
+  const withCentralApplicationConnectivity = process.env.CENTRAL_APPLICATION_CONNECTIVITY || false;
```

Comment:
You introduced two conventions. Names with `with` and without it. I personally would go for the `with` ones, so it'll be `withCentralApplicationConnectivity` and `WITH_CENTRAL_APPLICATION_CONNECTIVITY` everywhere

Reply:
I agree, but since the names are too long I shortened them to `withCentralAppConnectivity` and `WITH_CENTRAL_APP_CONNECTIVITY`.